src/surge-xt/gui/SurgeGUIEditorValueCallbacks.cpp

```diff
@@ -2350,6 +2350,63 @@ int32_t SurgeGUIEditor::controlModifierClicked(Surge::GUI::IComponentTagValue *c
                                 });
                         }
                     }
+                    break;
+                    case ct_amplitude_ringmod:
+                    {
+                        contextMenu.addSeparator();
+                        auto dt = p->deform_type;
+
+                        Surge::Widgets::MenuCenteredBoldLabel::addToMenuAsSectionHeader(
+                            contextMenu, "COMBINATOR MODE");
+
+                        contextMenu.addItem(
+                            "Ringmod (classic)", true, dt == CxorMode::RINGMOD, [this, p]() {
```

Comment:
So my suggestion for namese would simply be:

Default (multiply)
Continuous XOR 1...5

That's all!